src/CMakeLists.txt

```
@@ -120,13 +124,24 @@ elseif(CMAKE_CXX_COMPILER_ID STREQUAL "MSVC")
     )
 endif()
 
+if(NINTENDO_SWITCH)
+    set(${PROJECT}_C_CXX_EXTENSIONS ON)
+else()
+    set(${PROJECT}_C_CXX_EXTENSIONS OFF)
+endif()
+
+if(NOT (WIN32 OR APPLE OR NINTENDO_SWITCH))
+    find_package(X11 REQUIRED)
+    target_link_libraries(${EXECUTABLE} PRIVATE X11::X11)
+endif()
+
 set_target_properties(${EXECUTABLE}
```

@AGraber

Don't link to X11. This will get fixed soon by @erorcun.
```suggestion
endif()

set_target_properties(${EXECUTABLE}
```